website/blog/2024-1-18-while-youre-busy-fixing-vulnerabilities/item.md

```diff
@@ -0,0 +1,115 @@
+
+-   **Everybody has admin status** ([Sysdig reports](https://sysdig.com/2023-cloud-native-security-and-usage-report/) that 90% of accounts have excessive permissions!)
+-   **Passwords are stored in plain text somewhere** (probably to enable shared logins)
+
+These practices are risky for multiple reasons. If everyone has admin permissions, suddenly a hacker only needs one compromised account to gain access to everything and do a lot of damage. And we all agree that storing passwords in a spreadsheet is not secure, and makes it hard to isolate who has access to what, and why.
+
+Big, mature corporations tend to have more strict permissions. One of the blessings/curses about working in startups is that they are usually more flexible—employees have fewer limitations on the scope of their role, which can be great for collaboration and people taking initiative. But that fluidity can make it harder to keep tabs on who should have access to what. Being overly permissive with admin status also opens the door to [shadow IT](https://en.wikipedia.org/wiki/Shadow_IT).
+
+![](./phish2.png)
+
+### Integrate access reviews into planning cycles
+
+If your company is making quarterly or even annual plans at the moment, it’s a great time to join in that process. The worst time to sync up with engineering/product management teams to understand their access needs is in the middle of a sprint.
+
+If you’re partnering with engineering up front though, it’s much easier to work together to understand their plans, so as the security function you can keep an eye out for unusual requests. Slotting access management reviews into an existing process helps to prevent the “security as blocker” problem (which often leads to friction between security and dev teams).
+
+### Set up an Identity and Access Management solution
+
+You can automate away some of the more routine/tedious aspects of Identity and Access Management (IAM) by implementing an identity provider. With SSO (Single Sign-on), you can map specific tasks to certain roles, and then further map refined-access roles to other roles, based on what tools a person is likely to need in their specific position at the company. For example, if someone has access to AWS, they automatically have access to your database. By using an SSO, especially one with [RBAC](https://goauthentik.io/docs/user-group-role/access-control) (Role-Based Access Control), you are only concerned with the initial setup being correct versus constant, ongoing reviews.
+
+You can also set up alerts: these will send an email to the security team in the event that a new role is created so you can go and investigate.
+
+Full disclosure: we make (and use in-house) [authentik](https://goauthentik.io/), which is an identity provider. There is an open source version, so even if you’re strapped for cash you can still implement IAM. You can [check if authentik integrates with your application here](https://goauthentik.io/integrations/).
```

Comment:
> **We** are [authentik](https://goauthentik.io/), and our product is an RBAC-capable Single-Sign-On Identity Provider, and we use our own software to protect our assets.  Our [open-source version](https://github.com/goauthentik/authentik/) provides a full implementation of IAM. You can [check now](https://goauthentik.io/integrations/) to see if authentik integrates with the applications you use.

I don't know if you need the "Full disclosure."  I mean, I would hope people reading that would know where they were when they started.  :-)

Two things I just noticed: "Application" isn't a top-level thing in Docs, and they ought to be.  And it ought to be much easier to figure out how to add an application that's not on that list.  While it's obvious that creating an application requires some intimate knowledge of how an application is auth'd from the outside, it still feels baroque to read the different set-ups on the Integrations page...